src/css/style.css

```diff
@@ -1,151 +1,151 @@
 * {
-  margin: 0;
-  padding: 0;
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
 }
 :root {
-  --gray: #C4C4C4;
-  --black: #000000;
-  --white: #FFFFFF;
-  --gray-op-1: #C9D1D9;
-  --button-dark-blue: #001E66;
-  --button-light-blue: #76A4FF;
-  --input-border-blue: #3878F9;
-  --gray-op-2: rgb(139, 148, 158, 0.2);
-  --radius: 10px;
+	--gray: #c4c4c4;
+	--black: #000000;
+	--white: #ffffff;
+	--gray-op-1: #c9d1d9;
+	--button-dark-blue: #001e66;
+	--button-light-blue: #76a4ff;
+	--input-border-blue: #3878f9;
+	--gray-op-2: rgb(139, 148, 158, 0.2);
+	--radius: 10px;
```

Comment:
It's using tabs instead of spaces?

Reply:
Currently is using tabs, but, I'll change it to spaces